nym-vpn-core/crates/nym-statistics-api-client/src/error.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright 2025 - Nym Technologies SA <contact@nymtech.net>
+// SPDX-License-Identifier: GPL-3.0-only
+
+pub use nym_http_api_client::HttpClientError;
+
+#[derive(Debug, thiserror::Error)]
+pub enum StatisticsApiClientError {
+    #[error("failed tp create vpn api client")]
+    FailedToCreateVpnApiClient(#[source] HttpClientError),
```

Comment:
Nit: I think its' fair to assume that all errors are failures. So maybe removing `FailedTo` from them would be more concise and easier to read.

Reply:
Fixed 390b0dc1ade4e58168bd1ccc834a5bbda8fbd4b8